simulation/traffic_simulator/src/entity/vehicle_entity.cpp

```diff
@@ -71,11 +71,13 @@ auto VehicleEntity::getDefaultDynamicConstraints() const
 
 auto VehicleEntity::getDefaultMatchingDistanceForLaneletPoseCalculation() const -> double
 {
+  /// @note The lanelet matching algorithm should be equivalent to the one used in
+  /// EgoEntitySimulation::setStatus
   return std::max(
            vehicle_parameters.axles.front_axle.track_width,
            vehicle_parameters.axles.rear_axle.track_width) *
            0.5 +
-         1.0;
+         1.5;
```

Comment:
Please note in your comments that this value is a hard-coded parameter and the rationale for the value.